### PR TITLE
use variables to define colors in themes

### LINKED ISSUE #675: Use variables to defines colors in the themes
```CSS
* {
   -fx-bright: #eff0f1;
}

.text {
    -fx-fill: -fx-bright;
}
```

### PR DESCRIPTION
**Pull request by plata**
fixes #675

### CODE REVIEW

File: phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/breezeDark/main.less [hunk 1]
```diff
@@ -39,99 +45,99 @@
...
     -fx-text-fill: -fx-text-base-color;
 }
 
 .text-field {
     -fx-border-radius: 0;
     -fx-background-radius: 0;
-    -fx-background-color: #31363b;
+    -fx-background-color: @main-color;
     -fx-border-color: transparent;
     -fx-text-fill: -fx-text-base-color;
```
Comment (madoar) [inline phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/breezeDark/main.less]: I believe this should be replaced too.

Comment (plata) [inline phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/breezeDark/main.less]: Which one?

Comment (madoar) [inline phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/breezeDark/main.less]: `-fx-text-base-color` in `-fx-text-fill: -fx-text-base-color;`.
In addition there are a lot of similar occurences in the less files.

File: phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/breezeDark/main.less [hunk 2]
```diff
@@ -502,59 +508,59 @@
...
 /************************ step *************************/
 #panelForTopheader {
     -fx-padding: 2.5em;
-    -fx-background-color: -fx-background;
+    -fx-background-color: @background-color;
     -fx-border-color: transparent;
 }
 
 #header {
     -fx-border-color: -fx-focus-color;
```
Comment (madoar) [inline phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/breezeDark/main.less]: `-fx-focus-color` should be replaced too :)

File: phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less [hunk 3]
```diff
@@ -127,41 +132,41 @@
...
 .list-view {
-    -fx-background-color: #f2f1f0;
+    -fx-background-color: @background-color;
     -fx-border-color: #bdc3c7;
 }
 
 .list-cell {
-    -fx-background-color: #e7e5e4;
+    -fx-background-color: @main-color;
     -fx-text-fill: -fx-text-base-color;
```
Comment (madoar) [inline phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less]: Needs replacement

File: phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less [hunk 4]
```diff
@@ -127,41 +132,41 @@
...
 }
 
 .list-cell:even {
-    -fx-background-color: #f2f1f0;
+    -fx-background-color: @background-color;
     -fx-text-fill: -fx-text-base-color;
```
Comment (madoar) [inline phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less]: Needs replacement

File: phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less [hunk 5]
```diff
@@ -127,41 +132,41 @@
...
 }
 
 .list-cell:filled:hover, .list-cell:filled:selected:focused, .list-cell:filled:selected {
-    -fx-background-color: #df5e1e;
+    -fx-background-color: @focus-color;
     -fx-text-fill: -fx-text-base-color;
```
Comment (madoar) [inline phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less]: Needs replacement

File: phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less [hunk 6]
```diff
@@ -127,41 +132,41 @@
...
 }
 
 .text-field {
     -fx-border-radius: 0.3em;
     -fx-background-radius: 0.3em;
-    -fx-background-color: #e7e5e4;
+    -fx-background-color: @main-color;
     -fx-border-color: transparent;
     -fx-text-fill: -fx-text-base-color;
```
Comment (madoar) [inline phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less]: Needs replacement

File: phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less [hunk 7]
```diff
@@ -127,41 +132,41 @@
...
-    -fx-focus-color: #df5e1e;
+    -fx-focus-color: @focus-color;
 }
 
 .text-field:focused {
-    -fx-border-color: #df5e1e;
+    -fx-border-color: @focus-color;
     -fx-text-fill: -fx-text-base-color;
```
Comment (madoar) [inline phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less]: Needs replacement

File: phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less [hunk 8]
```diff
@@ -127,41 +132,41 @@
...
 }
 
 .text-field:hover {
-    -fx-border-color: #df5e1e;
+    -fx-border-color: @focus-color;
     -fx-text-fill: -fx-text-base-color;
```
Comment (madoar) [inline phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less]: Needs replacement

File: phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less [hunk 9]
```diff
@@ -404,10 +409,10 @@
 .searchBar {
     -fx-border-radius: 0;
     -fx-background-radius: 0;
-    -fx-background-color: #f2f1f0;
+    -fx-background-color: @background-color;
     -fx-border-color: transparent;
     -fx-text-fill: -fx-text-base-color;
```
Comment (madoar) [inline phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less]: Needs replacement

File: phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less [hunk 10]
```diff
@@ -416,7 +421,7 @@
 }
 
 .searchBar:hover {
-    -fx-border-color: #df5e1e;
+    -fx-border-color: @focus-color;
     -fx-text-fill: -fx-text-base-color;
```
Comment (madoar) [inline phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less]: Needs replacement

File: phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less [hunk 11]
```diff
@@ -629,16 +634,16 @@
 .consoleCommandType {
     -fx-border-radius: 0.7em;
     -fx-background-radius: 0;
-    -fx-background-color: #e7e5e4;
+    -fx-background-color: @main-color;
     -fx-border-color: transparent;
     -fx-text-fill: -fx-text-base-color;
```
Comment (madoar) [inline phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less]: Needs replacement

File: phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less [hunk 12]
```diff
@@ -629,16 +634,16 @@
...
 }
 
 .consoleCommandType:focused {
     -fx-border-radius: 0.7em;
     -fx-background-radius: 0;
-    -fx-background-color: #e7e5e4;
-    -fx-border-color: #df5e1e;
+    -fx-background-color: @main-color;
+    -fx-border-color: @focus-color;
     -fx-text-fill: -fx-text-base-color;
```
Comment (madoar) [inline phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less]: Needs replacement

File: phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/breezeDark/main.less (hunk 2 above)
Comment (plata) [inline phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/breezeDark/main.less]: what the heck... I searched for ": -".

**Pull request merged**